Testscripts/Linux/SetupRDMA.sh

```diff
@@ -52,6 +52,34 @@ function Verify_Result {
 	fi
 }
 
+function Upgrade_waagent {
+	# This is only temporary solution, WILL BE REMOVED as soon as 2.2.45 release in each image.
+	LogMsg "Starting waagent upgrade"
+	ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
in RHEL 8, python path is different.

Reply:
RHEL8 does not have python3 by default. I will fix it, but after the installation, it is there under /usr/bin/python3.

Reply:
fixed in 49405d6083f8541dc61a963bffa99fdfb2e38155